test/remote/gateways/remote_realex_test.rb

```diff
@@ -82,7 +82,7 @@ def test_realex_purchase_with_invalid_account
     assert_not_nil response
     assert_failure response
 
-    assert_equal '504', response.params['result']
+    assert_equal '506', response.params['result']
```

Comment:
This test was failing before I made any changes. I just updated it the actual error response code that I was getting back.

Reply:
Are you still getting that 506? On master, I'm getting a 504, so my test example fails when it checks for 506.

```
     82:     assert_not_nil response
     83:     assert_failure response
     84: 
  => 85:     assert_equal '506', response.params['result']
     86:     assert_match %r{no such}i, response.message
     87:   end
     88: 
test/remote/gateways/remote_realex_test.rb:85:in `test_realex_purchase_with_invalid_account'
<"506"> expected but was
<"504">
```

Reply:
We determined that I was getting a 504 because I had the default credentials. So, 506 should indeed be the appropriate response code expectation.